governance-exercises/VoteWizard_poll_exercise_04/Emergency Response Poll - Risk mitigation options - August 9, 2023.md

```diff
@@ -0,0 +1,72 @@
+discussion_link: https://forum.makerdao.com/t/...
+parameters:
+    input_format: single-choice
+    victory_conditions:
+        - { type : plurality }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Fix B
+   2: Fix B and Fix A
+start_date: 2023-08-09T10:00:00
+end_date: 2023-08-09T15:00:00
+---
+# Poll: Emergency Response Poll - Risk mitigation options - August 9, 2023
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of a Protocol Scope Facilitator. This [Emergency Governance Poll](https://mips.makerdao.com/mips/details/MIP24) will be active for 5 hours beginning on Wednesday, August 9 at 10:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+# Review
+
+The community can vote in this poll to express support for one of the following options that will be added to an emergency executive proposal to mitigate the risk of a potential exploit:
+
+* Fix B
+* Fix B and Fix A
+
+Please review the descriptions of the two possible fixes:
+
+## Fix A
+
+- Fix A prevents the high-severity exploit immediately upon the executive vote passing.
+- Fix A is likely to incur losses to MKR holders of approximately 20 million DAI over two weeks.
+
+## Fix B
+
+- Fix B closes the high-severity exploit approximately two weeks after the executive vote passes.
+- Fix B has no risk of incurring financial loss to MKR holders.
+
+## Additional Information
+
+[Information further elaborated and assessed by the ecosystem expert]
+
+Note that Fix B will be included in the executive regardless of the approved option.
+
+Please review the discussion [thread](https://forum.makerdao.com/t/...) to inform your position on this issue before voting.
+
+# Outcomes
+
+**If the votes for the 'Fix B' option exceed the votes for the 'Fix B and Fix A' option then the following actions will be taken:**
+* Fix B will be added to the next executive to mitigate the potential exploit.
+* It is expected that this Executive Vote will take place within one hour of this poll passing, absent external factors.
+* If the Executive Vote passes, then this change will become active in the Maker Protocol after the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay) has expired.
+
+**If the votes for the 'Fix B and Fix A' option exceed the votes for the 'Fix B' option then the following actions will be taken:**
+* Fix B and Fix A will be added to the next executive to mitigate the potential exploit.
+* It is expected that this Executive Vote will take place within one hour of this poll passing, absent external factors.
+* If the Executive Vote passes, then this change will become active in the Maker Protocol after the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay) has expired.
```

Comment:
Not true in the case of fix A. It's specifically noted that it will come into effect the instant the executive vote passes.